File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.java
import java.io.*;
import java.util.*;


public class Main{
    public static void main(String[] args)throws Exception{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine());
        int[] arr = new int[N];
        for(int i=0; i<N; i++){
            arr[i] = Integer.parseInt(br.readLine());
        }
        Arrays.sort(arr);
        for(int i=0; i<N;i++){
            System.out.println(arr[i]);
        }
    }
}

File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let inputLine = 0;
let N = 0;
let arr = [];
rl.on("line", (line) => {
  if (inputLine == 0) {
    N = parseInt(line);
  }
  if (inputLine > 0 && inputLine <= N) {
    arr.push(line);
  }
  if (inputLine == N) {
    arr.sort((a, b) => a - b);
    for (let i = 0; i < arr.length; i++) {
      console.log(arr[i]);
    }
    process.exit();
  }
  inputLine++;
});
